src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -195,3 +198,62 @@
               :on-press            (fn []
                                      (rf/dispatch [:hide-bottom-sheet])
                                      (when on-press (on-press)))}]))]])))
+
+(defn get-reaction-authors-accessibility-id
+  [reaction-type]
+  (-> reaction-type
+      constants/reactions
+      name
+      (str "-reaction-count")))
+
+(defn reaction-authors
+  [reaction-authors show-reaction-author-list?]
+  (let [selected-tab (reagent/atom (first (keys reaction-authors)))]
+    [:f>
```

Comment:
It's worth pointing out we've had recent discussions about the proper way to use functional components, and this pattern we've been using in the codebase is *dangerous*, in the sense that it can lead to unexpected side-effects and cause performance degradation.

https://github.com/status-im/status-mobile/pull/15686/files#diff-b5e21cecfbe129756a13073f455833097d90938c8367957243f550fe8d76e567R39

Reply:
Done.
Remembered something like this when writing this. But didn't find a reference to the document.
Thanks for sharing!